Geometry/HcalCommonData/src/HcalDDDRecConstants.cc

```diff
@@ -595,6 +595,20 @@ void HcalDDDRecConstants::specialRBXHBHE(const std::vector<HcalDetId>& idsOld,
   }
 }
 
+bool HcalDDDRecConstants::specialRBXHBHE(bool tobemerged,
+					 std::vector<HcalDetId>& ids) const {
+  ids.clear();
```

Comment:
@bsunanda - why do you need to clear the vector here? You fill an empty newly created vector from the way it is used in the test. Note, that the capacity (memory usage) of a vector will never decrease, even if you call the clear method, unless you use the swap method.